xla/stream_executor/gpu/gpu_cudamallocasync_allocator.cc

```diff
@@ -211,69 +231,28 @@ GpuCudaMallocAsyncAllocator::GpuCudaMallocAsyncAllocator(
     }
   }
 
-  // Set read/write access to all GPUs.
```

Comment:
Why completly remove it?
It should be at least part of the description, or in a different PR.
This is a significant changes.

Reply:
I did not figure out a case where async allocator needs to access other device's pool, and actually this is contradict to the assumption that allocator is per GPU device. I am trying to remove it to see if any tests fails, but seems that all XLA tests have passed.  

If TF requires such a feature, this looks quite strange to me.